docs/api/spinner.md

```diff
@@ -37,19 +37,19 @@ import Colors from '@site/static/usage/v7/spinner/theming/colors/index.md';
 
 <Colors />
 
-### CSS Custom Properties
+### Styling
 
-import CSSProps from '@site/static/usage/v7/spinner/theming/css-properties/index.md';
+You may use custom CSS to style the spinner. For example, you can resize the spinner by setting the width and height.
```

Comment:
Suggestion to change: `You can resize the spinner by setting the width and height.`